.all-contributorsrc

```diff
@@ -2,7 +2,11 @@
   "projectName": "all-contributors",
   "projectOwner": "kentcdodds",
   "files": [
```

Comment:
FYI we actually have an undocumented maximum on the bot of 5 files here. See: https://github.com/all-contributors/all-contributors-bot/blob/master/src/ContentFiles/index.js#L31